extensions/index.md

```diff
@@ -83,7 +83,8 @@ Once submitted, one of JSON:API's editors will review your profile to check that
 
 1. follows the template above; 
 2. is specified precisely enough to enable interoperable implementations;
-3. complies with the JSON:API spec and its [requirements for profiles](/format/1.1/#profiles-authoring); and
+3. complies with the JSON:API spec and its [requirements for profiles](/format/1.1/#profiles-authoring);
+3. follows JSON:API's [recommended naming conventions](https://jsonapi.org/recommendations/#naming); and
 4. wouldn't cause any problems were it to become widely adopted. 
```

Comment:
s/4/5/

Reply:
Maybe these should all just be `1`. Markdown (at least Github's flavor) should auto-increment the numbers.

Reply:
Haha, @wimleers you're giving me too much credit :P I just messed up the numbering. I'm fine with using all `1`s, but we should probably be consistent about that. Since I think we use proper numbering elsewhere, I've just fixed the numbering here for now.